tests/Integration/PositionstackGeocoderTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Tests\Integration;
+
+use Tests\TestCase;
+use App\Services\Geocoder\PositionstackGeocoder;
+
+class PositionstackGeocoderTest extends TestCase {
+
+    public function testGeocode() {
+
+        // Skip test if API key not configured
+        // if (config('positionstack.api_key') === null) {
+        //     return;
+        // }
```

Comment:
Just let it fail, otherwise you can think "everything is fine" while it isn't

Reply:
The thing is, I would expect this test to fail in the CI pipeline, since it doesn't have the Positionstack API key (yet). But... it didn't fail last time. Strange...